### PR TITLE
Update Injective Chain Info

### PR DESCRIPTION
**Pull request by nmarcetic**
Signed-off-by: Nikola Marcetic <n.marcetic86@gmail.com>

This PR updates [Injectiveprotocol](http://injectiveprotocol.com/) chain data.

### CODE REVIEW

File: injective/chain.json [hunk 1]
```diff
@@ -7,28 +7,96 @@
...
+            "id": "30e81e1b785eaf82d2260e932251c02c95787740",
+            "address": "30e81e1b785eaf82d2260e932251c02c95787740@86.109.11.215:26656",
+            "provider": "injectiveprotocol"
+        }
+    ]
+    },
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://public.grpc.injective.network",
```
Comment (sunnya97) [inline injective/chain.json:70]: This should be moved to a "grpc" category, not in "rpc"

**Comment by sunnya97** [comment]
What is your SLIP44 path?

**Comment by sunnya97** [comment]
Please also add an assetlist

**Comment by nmarcetic** [comment]
Thank you @sunnya97 I will add `assetlist` ASAP in dedicated PR.

**Comment by sunnya97** [comment]
@nmarcetic check out #143.  Does the one there look good?

**Comment by nmarcetic** [comment]
> @nmarcetic check out #143. Does the one there look good?

Yes, thank you @sunnya97

**Pull request merged**